source/making_payments/index.html.md.erb

```diff
@@ -77,37 +100,36 @@ datastore mapped to the payment ID just after you create a payment.
  * not paid because of a technical error
```

Comment:
There’s also a couple of other reasons:

- their payment being declined (not really a technical error)
- the user cancelled the payment

Reply:
Are these distinct from the matter covered in this bullet(?):

> * not paid because their card was rejected or they selected cancel